Sources/StreamChat/Utils/InternetConnection/Error+InternetNotAvailable.swift

```diff
@@ -40,4 +40,8 @@ extension Error {
         }
         return false
     }
+
+    var isSocketNotConnectedError: Bool {
+        has(parameters: (NSPOSIXErrorDomain, 57))
```

Comment:
Is this enough to check for socket errors? I also see https://developer.apple.com/documentation/foundation/nsstreamsockserrordomain.

Reply:
I guess it's fine, since it's only used for logging.